google-cloud-pubsub/lib/google/cloud/pubsub/subscription.rb

```diff
@@ -553,19 +542,16 @@ def seek snapshot
         #
         #   sub.policy do |p|
         #     p.add "roles/owner", "user:owner@example.com"
-        #   end # 2 API calls
+        #   end
         #
         def policy force: nil
-          @policy = nil if force || block_given?
-          @policy ||= begin
-            ensure_service!
-            grpc = service.get_subscription_policy name
-            Policy.from_grpc grpc
-          end
-          return @policy unless block_given?
-          p = @policy.deep_dup
-          yield p
-          self.policy = p
+          puts "DEPRECATED: 'force' in Subscription#policy" unless force.nil?
```

Comment:
If `force` is removed we don't need this. But for Storage we should use `warn` instead of `puts`.